### PR TITLE
PR - Milestone 3 - Users and Posts

### PR DESCRIPTION
**Pull request by vieiramanda11**
Hello.

We used devise to create mailer confirmation so in order to test our code it's necessary to confirm after you sign up in a link in your terminal when you run the server.

Thank you for your review,

Amanda and Ansar.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  validates :first_name, presence: true, length: { in: 5..20 }
+  validates :last_name, presence: true, length: { in: 5..20 }
```
Comment (adriaanbd) [inline app/models/user.rb]: Setting a minimum length of 5 characters is too much in my opinion. What about names like "Joe", "John", "Joe" or even "Al"? These users won't be able to sign up with the above-mentioned constraint. I'd strongly suggest to remove the length constraint to a minimum of two characters [`OPTIONAL`].

File: app/views/shared/_feed.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+<% @posts.each do |p| %>
```
Comment (adriaanbd) [inline app/views/shared/_feed.html.erb]: Posts need to always display post content and creator/author according to requirements [`REQUIRED`]

File: app/views/layouts/_navigation.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+
+<nav class="navbar navbar-default ">
+  <div class="container-fluid navigation">
+    <div class="navbar-left navigation-left">
+      <h2><%= link_to 'FACEBOOK', root_path %></h2>
+    </div>
+    <%if user_signed_in?%>
+      <ul class="nav navbar-nav navbar-right header">
+       <li><%= link_to "Post", "#" %></li>
```
Comment (adriaanbd) [inline app/views/layouts/_navigation.html.erb]: I clicked here thinking it'll take me to `posts#index` view. What do you plan on using this link for when it's finally functional?

File: app/views/shared/_feed.html.erb (hunk 2 above)
Comment (adriaanbd) [reply app/views/shared/_feed.html.erb]: Posts also need to be displayed in a timeline manner, according to requirements, with most recent posts displayed first. Currently, oldest post is first, take a look:

image

[`REQUIRED`]

**Comment by vieiramanda11** [comment]
All corrections and suggestions were implemented!

**Pull request merged**